modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/AMQPUtils.java

```diff
@@ -387,18 +389,30 @@ public static InboundExchangeEvent createAndesExchange(Exchange exchange) {
      * @param routingKey routing key
      * @return InboundBindingEvent binding event that wrap AndesBinding
      */
-    public static InboundBindingEvent createAndesBinding(Exchange exchange, AMQQueue queue, AMQShortString routingKey) {
+    public static InboundBindingEvent createAndesBinding(Exchange exchange, AMQQueue queue, AMQShortString routingKey)
+            throws AMQInternalException {
         /**
          * we are checking the type of exchange to avoid the confusion
          * <<default>> exchange is actually a direct exchange. No need to keep two bindings
          */
         //TODO: extend to other types of exchanges
         String exchangeName = "";
+        log.info("Create andes binding for AMQ Queue : "+ queue+ ", Routing Key is "+routingKey);
```

Comment:
Should this be an info log? Will it be useful to know (or help in debugging) each and every binding created in QPID?